frontend/src/pages/Workflows/WorkflowItem.js

```diff
@@ -320,12 +333,19 @@ const renderActionButtons = (
   workflowSortEnabled,
   status,
   showAdditionalData,
-  additionalData
+  additionalData,
+  idsPermissionsUnassigned,
+  id
 ) => {
   const additionalDataDisabled = _isEmpty(additionalData) || workflowSortEnabled;
   const editDisabled = !canEditWorkflow || workflowSortEnabled;
   const permissionsDisabled = !canListWorkflowPermissions || workflowSortEnabled;
-
+  const hideBadge =
+    idsPermissionsUnassigned.find(el => el === id) === undefined
+      ? true
+      : workflowSortEnabled || permissionsDisabled
+      ? true
+      : false;
```

Comment:
maybe rewrite as if-statement for better readability

Reply:
additionally: you can put `idsPermissionsUnassigned.find(el => el === id) === undefined` into a separate variable, e.g. `workflowitemAssigneeChanged` so the if-statement is more readable